applications/dashboard/settings/class.hooks.php

```diff
@@ -654,6 +654,22 @@ public function gdn_dispatcher_appStartup_handler($sender) {
             }
         }
         $this->checkAccessToken();
+
```

Comment:
@tburry You are gonna have to review this part.

Reply:
This is how you [bypass a CSRF ban](https://github.com/vanilla/hootsuite/blob/master/addons/hootsuite/controllers/hootsuiteapicontroller.php#L156) for API controllers.

Reply:
I needed to merge in the update from #5990. Works, now!